modules/enterprise/server/jar/src/main/java/org/rhq/enterprise/server/resource/group/LdapGroupManagerBean.java

```diff
@@ -638,8 +638,11 @@ private void executeGroupSearch(String filter, Set<Map<String, String>> groupDet
             }
             //
             Map<String, String> entry = new HashMap<String, String>();
-            String name = (String) si.getAttributes().get("cn").get();
-            name = name.trim();
+            Attribute nameAttribute = si.getAttributes().get("cn");
+            if (nameAttribute == null) {
+                continue;
+            }
+            String name = ((String) nameAttribute.get()).trim();
```

Comment:
Just a small question.. can the nameAttribute be non-null but the value itself from ``.get()`` be null? Or should it be checked in the above ``nameAttribute == null || nameAttribute.get() == null`` ?

Reply:
I'm not sure. 
I would think that if we already have an value it will be non null.
I'll add the extra test to be sure of that.